test/TritonGPU/amd/accelerate-amd-matmul-mfma-gfx950.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: triton-opt %s -split-input-file --tritonamdgpu-accelerate-matmul='arch-generation-name=gfx950 matrix-instruction-size=0' | FileCheck %s --check-prefixes CHECK
+// RUN: triton-opt %s -split-input-file --tritonamdgpu-accelerate-matmul='arch-generation-name=gfx950 matrix-instruction-size=0' -tritongpu-remove-layout-conversions | FileCheck %s --check-prefixes CHECK
```

Comment:
why running `tritongpu-remove-layout-conversions` here as this is meant to test something completely different? That defeats the purpose of unit testing

Reply:
Removed